### DISCUSSION TITLE
List in "Track thru Files in list"

### DISCUSSION THREAD

**Discussion by RionIwasaki** [WeirongChen/DeepEdge#7] [Q&A]
Hello Weirong,
I have a question about list in "Track thru Files in list". I thought this option allowed me to track all files in a file, only to realize that I needed a file list to apply auto-tracking across multiple files. I wonder what a file list should look like. Do you have any examples? 
Thank you in advance.

**Comment by WeirongChen** [comment]
Hi Rion, 
Thanks for posting the question. 
When you click File -> Load Video or Load Video + Contour (+TextGrid),
You can select more files by press "Ctrl" or "Shift", thus creating an internal "FileList" in DeepEdge. 

After you selected (>1) video files, you can click File->Show/Selected from FileList to check the FileList, and/or switch between the files in the Filelist to process.  This function will also show you how many frames have been contoured for each file in the FileList. 

The function "Track thru Files in list" will run through all files in that FileList. 

Hope this helps.

**Reply by RionIwasaki** [reply]
Thank you for the answer, Weirong! I see. So, selecting multiple video files create an internal "FileList". I get it. I didn't have time to test it today, but I will tomorrow and see how it goes.

**Reply by RionIwasaki** [reply]
Indeed, it has worked! I was able to process two files by selecting these two files with "Ctrl". I was able to see the content of the FileList before processing. While it took a while to complete tracking of both files as both files are about 4 to 5 minutes, all frames of both files are now tracked. Thank you for the instruction, Weirong!
